ckanext/harvest/logic/action/update.py

```diff
@@ -491,6 +496,88 @@ def harvest_jobs_run(context, data_dict):
 
     return []  # merely for backwards compatibility
 
+def send_error_mail(context, job_obj):
+
+    sql = 'select name from package where id = :source_id;'
+
+    model = context['model']
+
+    q = model.Session.execute(sql, {'source_id': job_obj.source.id})
+
+    for row in q:
+        harvest_name = str(row['name'])
+
+    ckan_site_url = config.get('ckan.site_url')
+    job_url = ckan_site_url + '/harvest/' + harvest_name + '/job/' + job_obj.id
+
+    msg = 'This is a failure-notification of the latest harvest job (' + job_url + ') set-up in ' + ckan_site_url + '.\n\n'
+
+    sql = '''select g.title as org, s.title as job_title from member m
```

Comment:
Depending on your authorisation setup not all harvest sources might belong to an organization (for instance if `ckan.auth.create_unowned_dataset = true`). You should consider that when building the output.